src/doc/book/mutability.md

```diff
@@ -55,6 +55,8 @@ fn foo(mut x: i32) {
 # }
 ```
 
+Note that here, the `x` is mutable, but not the `y`.
```

Comment:
I can't see that in the master. But one easy fix is to change "Note that here" to "Here".